tide_core.module

```diff
@@ -58,6 +60,85 @@ function tide_core_entity_operation(EntityInterface $entity) {
       }
     }
   }
+  if (\Drupal::moduleHandler()->moduleExists('scheduled_transitions') && $entity->getEntityType()->hasLinkTemplate(ScheduledTransitionsRouteProvider::LINK_TEMPLATE_ADD)) {
+    $routeName = ScheduledTransitionsRouteProvider::getScheduledTransitionRouteName($entity->getEntityType());
+    $url = Url::fromRoute($routeName, [$entity->getEntityTypeId() => $entity->id()]);
+    $user = \Drupal::currentUser();
+    if (TRUE === $url->access($user)) {
+      $operations['scheduled_transitions'] = [
+        'title' => \t('Scheduled publishing'),
```

Comment:
@vincent-gao `t()` is a function, no need to use `\` namespace.

Reply:
fixed. thanks @sonnykt